src/sentry/static/sentry/app/views/settings/organizationMembers/index.jsx

```diff
@@ -345,4 +344,8 @@ const SubText = styled('p')`
   font-size: 15px;
 `;
 
+const InviteButton = styled(Button)`
+  height: 36px;
```

Comment:
Should we instead use `size="large"` on the button?

Reply:
The default size might be okay, but is a little on the large side. I can keep this as is for now